ddtrace/tracer.py

```diff
@@ -27,7 +27,7 @@ class Tracer(object):
         from ddtrace import tracer
         trace = tracer.trace("app.request", "web-server").finish()
     """
-    DEFAULT_HOSTNAME = 'localhost'
+    DEFAULT_HOSTNAME = environ.get('DATADOG_TRACE_AGENT_HOSTNAME', 'localhost')
```

Comment:
In the current state, this is something we can use to address the problem you're mentioning, but in the near future we're going to refactor this part so that any settings can be defined via env vars.

Another thing is that `DATADOG_TRACE_AGENT_HOSTNAME` is a deprecated name and it should be `DD_TRACE_AGENT_HOSTNAME`. For this reason we have this `get_env()` call ([ref](https://github.com/carium-inc/dd-trace-py/blob/b4817b8f77ae653947006daedeea1782e0698c74/ddtrace/utils/formats.py#L6-L31)) that accepts but deprecate `DATADOG_*` in favor of `DD_*`.

Unfortunately, we still need to do the change globally in all integrations and settings. Because it will take a bit of time, we can approve and merge your PR now, while thinking of a re-work a bit later.

Thanks a lot for the contribution!

Reply:
We could definitely update it to the get_env() call and use the "modern" variable naming.  Let us know if you want us to revisit...

Reply:
No worries this is fine that way. We need to do a global refactoring here so it's clear to new contributors how to retrieve environment variables. For now I guess it's only a matter of shipping a new release.

We'll revisit it later (no breaking changes will be added).